From e8ed0a25e03eda9d6914e54946c4cf027bed4a60 Mon Sep 17 00:00:00 2001
From: Huang Chenyi <b44402@freescale.com>
Date: Fri, 12 Jul 2013 14:50:47 +0800
Subject: [PATCH 18/37] ENGR00270791 Change time format and the canvas's
 background color

Change time format from "yyyy-MM-dd HH:mm:ss:SSS" to "mm:ss:SSS".
Change the background color of canvas from black to  translucent.

Signed-off-by: Huang Chenyi <b44402@freescale.com>
---
 packages/SystemUI/src/com/android/systemui/LoadSystemTime.java | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java b/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java
index fee5856..eb75849 100644
--- a/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java
+++ b/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java
@@ -31,6 +31,7 @@ import android.view.Gravity;
 import android.view.View;
 import android.view.WindowManager;
 
+import android.graphics.PixelFormat;
 import android.graphics.Color;
 import android.graphics.PorterDuff.Mode;
 import android.graphics.PorterDuff;
@@ -85,6 +86,7 @@ public class LoadSystemTime extends Service {
             mAddedPaint.setARGB(255, 128, 255, 128);
 
             surfaceHolder = this.getHolder();
+            getHolder().setFormat(PixelFormat.TRANSLUCENT);
             surfaceHolder.addCallback(this);
         }
 
@@ -146,7 +148,7 @@ public class LoadSystemTime extends Service {
     public void onCreate() {
         super.onCreate();
         mIndex = 0;
-        mPattern = "yyyy-MM-dd HH:mm:ss:SSS";
+        mPattern = "mm:ss:SSS";
         mSdf = new SimpleDateFormat(mPattern);
         mTimeList = new ArrayList<String>();
         for(int i = 0; i < mLineNumber; i++ ){
-- 
1.8.0

